src/app/store/effects/upload.effects.ts

```diff
@@ -147,9 +145,9 @@ export class UploadEffects {
           this.fileVersionInput.value = '';
           return of(new SnackbarErrorAction('VERSION.ERROR.GENERIC'));
         })
-      );
-    })
-  );
+      )
+      .subscribe(noop);
```

Comment:
I think we don't need "tap" in this case, moving the content to the "subscribe" would make more sense

Reply:
tap is needed as a side effect in case the dialog is canceled to clear the input value otherwise, the change event won't trigger when we then select the same file to upload

Reply:
but I can refactor further and have `subscribe` with a meaning